coreneuron/nrniv/nrnoptarg.cpp

```diff
@@ -112,7 +112,7 @@ static param_str param_str_args[] = {
 
 static void graceful_exit(int);
 
-static ez::ezOptionParser* opt;
+static ez::ezOptionParser* opt = NULL;
```

Comment:
this doesn't hurt but `static pointer is always initialised to null` by default.